core/services/ocr2/plugins/llo/integration_test.go

```diff
@@ -1493,17 +1491,11 @@ channelDefinitionsContractFromBlock = %d`, serverURL, serverPubKey, donID, confi
 				}
 			}
 			if finished >= nNodes {
-				stopOnce.Do(func() {
-					// Stop all nodes, close the channel
-					// This helps transmissions have a chance to complete (but
-					// doesn't ensure it; libocr cancels the transmit context
-					// immediately on stop signal)
-					// Loop will exit once all packets are consumed
-					for _, node := range nodes {
-						require.NoError(t, node.App.Stop())
-					}
-					close(packets)
-				})
+				for _, node := range nodes {
+					require.NoError(t, node.App.Stop())
```

Comment:
Let's assert so any remaining nodes are not abandoned if one fails to stop cleanly:
```suggestion
					assert.NoError(t, node.App.Stop())
```

Reply:
golangci-lint seems to like require better.

<checkstyle version="5.0">
  <file name="core/services/ocr2/plugins/llo/integration_test.go">
    <error column="6" line="1495" message="require-error: for error assertions use require" severity="error" source="testifylint"></error>
  </file>
</checkstyle>

Reply:
huh that seems like a bad rule
You could just collect the errors via `errors.Join` and then do one require after :shrug:

Reply:
We can't go back to the require, because leaking nodes will cause data races too